syft/messaging/plan.py

```diff
@@ -477,12 +487,14 @@ def replace_ids(
             # for every message of the plan
             for j, msg in enumerate(self.readable_plan):
                 # look for the old id and replace it with the new one
-                self.readable_plan[j] = Plan._replace_message_ids(
-                    obj=msg,
-                    change_id=from_ids[i],
-                    to_id=to_ids[i],
-                    from_worker=from_worker,
-                    to_worker=to_worker,
+                self.readable_plan[j] = list(
```

Comment:
what's the benfit of casting it into a list?

Reply:
I don't see any, I think that the only difference is that the list could be modified but it's not done anywhere.
@iamtrask, can I remove the conversion?